enterprise/glassfish.common/src/org/netbeans/modules/glassfish/common/ServerDetails.java

```diff
@@ -480,6 +459,9 @@ public static int getVersionFromInstallDirectory(File glassfishDir)  {
                 case GF_7_0_4:   return GLASSFISH_SERVER_7_0_4.getVersion();
                 case GF_7_0_5:   return GLASSFISH_SERVER_7_0_5.getVersion();
                 case GF_7_0_6:   return GLASSFISH_SERVER_7_0_6.getVersion();
+                case GF_7_0_7:   return GLASSFISH_SERVER_7_0_7.getVersion();
+                case GF_7_0_8:   return GLASSFISH_SERVER_7_0_8.getVersion();
+                case GF_7_0_9:   return GLASSFISH_SERVER_7_0_9.getVersion();
```

Comment:
can this switch be eliminated by putting `GLASSFISH_SERVER_` enums into `GF_7_` enums during construction?